app/src/main/res/layout/fragment_onboarding_signup.xml

```diff
@@ -74,6 +87,7 @@
                 android:layout_height="wrap_content"
                 android:layout_margin="60dp"
                 android:text="Continue"
+                android:onClick="@{() -> signUpCallback.invoke()}"
```

Comment:
If I also take Shalom's advice and refactor so there's no need for a `context`, we could even call `viewModel.createUser` directly here 👍

Reply:
💯